src/common-principles.md

```diff
@@ -409,20 +409,22 @@ datasets and non-compliant derivatives.
 
 ### Imaging files
 
-All imaging data MUST be stored using the NIfTI file format. We RECOMMEND using
-compressed NIfTI files (.nii.gz), either version 1.0 or 2.0. If using compressed files,
-the gzip header SHOULD lack source filenames and timestamps. Imaging data SHOULD
-be converted to the NIfTI format using a tool that provides as much of the NIfTI
-header information (such as orientation and slice timing information) as
-possible. Since the NIfTI standard offers limited support for the various image
-acquisition parameters available in DICOM files, we RECOMMEND that users provide
-additional meta information extracted from DICOM files in a sidecar JSON file
-(with the same filename as the `.nii[.gz]` file, but with a `.json` extension).
-Extraction of BIDS compatible metadata can be performed using [dcm2niix](https://github.com/rordenlab/dcm2niix)
-and [dicm2nii](https://www.mathworks.com/matlabcentral/fileexchange/42997-xiangruili-dicm2nii)
-DICOM to NIfTI converters. The [BIDS-validator](https://github.com/bids-standard/bids-validator)
-will check for conflicts between the JSON file and the data recorded in the
-NIfTI header.
+All imaging data MUST be stored using BIDS-compatible file format, specified 
+in the respective section. The conversion from original imaging data format
+to BIDS-compatible one must be performed using a tool that conserves as much
```

Comment:
```suggestion
to BIDS-compatible one, if it is required, must be performed using a tool that conserves as much
```
Some datatypes have formats that do not need conversion from their source format.

Reply:
Agreed

Reply:
Strange I cannot just commit those suggestions. That's usually possible from the github UI. @sappelhoff  ever seen that before?

Reply:
Also it seems that maintainers cannot edit the PR...

On other PRs we have this:

![image](https://github.com/bids-standard/bids-specification/assets/6961185/2f1dc3d3-8c9c-42a5-befb-183bbb799fce)

but not this one...

Reply:
Maybe my fault, I remembered there was an option to allow maintainers to commit, but I can't find it anymore.